akka-docs/src/main/paradox/additional/rolling-updates.md

```diff
@@ -178,19 +178,19 @@ If you need to change any of the following aspects of sharding it will require a
  
 ### Cluster configuration change
 
-* A full restart is required if you change the [SBR strategy](https://doc.akka.io/docs/akka/current/split-brain-resolver.html#strategies)
+* A full restart is required if you change the [SBR strategy](https://doc.akka.io/libraries/akka-core/current/split-brain-resolver.html#strategies)
 
 ### Migrating from PersistentFSM to EventSourcedBehavior
 
-If you've migrated from `PersistentFSM` to `EventSourcedBehavior` (See the [Akka 2.8 migration guide](https://doc.akka.io/docs/akka/2.8/persistence-fsm.html#migration-to-eventsourcedbehavior))
+If you've migrated from `PersistentFSM` to `EventSourcedBehavior` (See the [Akka 2.8 migration guide](https://doc.akka.io/libraries/akka/2.8/persistence-fsm.html#migration-to-eventsourcedbehavior))
```

Comment:
libraries/akka-core/ here, or is there is a reason for keeping the old versions as libraries/akka/ ?

Reply:
I was thinking "change as little as possible", but don't have a strong opinion. This PR only updates links to `current` (thus future) versions.

Reply:
seems more difficult to keep track of different links for before and after a certain point. since we are anyway adding redirects (or symlinks) I think it's better to change all of them